README.md

```diff
@@ -1,44 +1,233 @@
+
+### [Docker Compose](https://docs.docker.com/compose/install/)
+To check your docker-compose installation, please run the following command. It uses the "hello-world" image from the previous section:
+``` shell
+docker-compose -f - up <<EOF
+version: "3.9"
+services:
+  hello-world:
+    image: hello-world
+EOF
+```
+After executing the command, you should again see the message starting with "Hello from Docker!".
+
+### [systemd](https://systemd.io/)
+You shouldn't need to install it yourself. If systemd is not available on your system you should get another system.
+To check if systemd is available on your system, please execute
+
+``` shell
+systemctl --version
+```
+
+## Getting Started
+
+If your system passed all checks from ["Requirements" section], you are now ready to download the bridgehead.
+
+First, clone the repository to the directory "/srv/docker/bridgehead":
+
+``` shell
+sudo mkdir /srv/docker/;
+sudo git clone https://github.com/samply/bridgehead.git /srv/docker/bridgehead;
+```
+
+Next, you need to download the cooperatively created site-configuration to the directory "/srv/docker/bridgehead/site-config":
+
+``` shell
+sudo git clone <link-to-sites-configuration-repository> /srv/docker/bridgehead/site-config;
+```
+
+After this, you need to install the systemd units. This is currently done with the [install-bridgehead.sh script](./install-bridgehead.sh).
+
+You can execute it with
+``` shell
+sudo ./install-bridgehead.sh
+```
+
+Finally, you need to configure your sites secrets. These are places as configuration for each bridgeheads system unit. Refer to the section for your specific project:
+
+### DKTK/C4
+
+You can create the site specific configuration with: 
+
+``` shell
+sudo systemctl edit bridgehead@dktk.service;
+```
+
+This will open your default editor allowing you to edit the docker system units configuration. Insert the following lines in the editor and define your machines secrets. You share some of the ID-Management secrets with the central patientlist (Mainz) and controlnumbergenerator (Frankfurt). Refer to the ["Configuration" section]() for this.
+
+``` conf
+[Service]
+Environment=HOSTIP=
+Environment=HOST=
+Environment=HTTP_PROXY_USER=
+Environment=HTTP_PROXY_PASSWORD=
+Environment=HTTPS_PROXY_USER=
+Environment=HTTPS_PROXY_PASSWORD=
+Environment=CONNECTOR_POSTGRES_PASS=
+Environment=STORE_POSTGRES_PASS=
+Environment=ML_DB_PASS=
+Environment=MAGICPL_API_KEY=
+Environment=MAGICPL_MAINZELLISTE_API_KEY=
+Environment=MAGICPL_API_KEY_CONNECTOR=
+Environment=MAGICPL_MAINZELLISTE_CENTRAL_API_KEY=
+Environment=MAGICPL_CENTRAL_API_KEY=
+Environment=MAGICPL_OIDC_CLIENT_ID=
+Environment=MAGICPL_OIDC_CLIENT_SECRET=
+```
+
+For the C4 project, you need to additionally set the "LDM_URL_BASE" setting in the configuration table to null. You need to execute this after starting the bridgehead system unit for c4:
```

Comment:
We should separate DKTK from C4. Also in the long term the LDM_URL will change in DKTK as well